src/dotnet/APIView/APIViewWeb/LeanModels/ChangeHistory.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace APIViewWeb.LeanModels
 {
+    [System.Text.Json.Serialization.JsonConverter(typeof(System.Text.Json.Serialization.JsonStringEnumConverter))]
```

Comment:
nit: can you import `System.Text.Json.Serialization.` so you don't have to add the whole thing?

Reply:
Am also using Newton Soft on the class. Am doing this to avoid class name conflict.